src/org/omegat/core/data/ProjectProperties.java

```diff
@@ -87,7 +87,7 @@ protected ProjectProperties() {
      * Default constructor to initialize fields (to get no NPEs). Real values
      * should be applied after creation.
      */
-    public ProjectProperties(File projectDir) throws Exception {
+    public ProjectProperties(File projectDir) {
```

Comment:
It should be checked that caller may have a try block, and now constructor does not throw exception.
We should update callers to remove a try block or a catch clause.

Reply:
The constructor here is called by 58 places. Many are located in language modules and spellchecker modules. Otherwise, there are 20 in the test code. Does the change affect 3rd party plugins?